### ISSUE TITLE
Doens't work on arch

### ISSUE THREAD

**Issue by ralyodio** [inspectdev/inspect-issues#137]
- Inspect version: 0.6.9
- Electron: 13.1.1
- OS version and architecture: linux x64 5.16.11-2-MANJARO
- Language: en_US.UTF-8

I'm running Manjaro and the aur package installed. I got promopted on iPhone to trust. but nothing shows in inspector app on my Linux desktop. There is a screen about downloading iTunes for windows, but I don't use Windows. I'm on Linux.

**Comment by auchenberg** [comment]
@ralyodio Thanks for reporting.

We don't own the submitted packaged on https://aur.archlinux.org/packages/inspect#comment-855662. Looks like it's maintained by a community member.

Inspect is not tested on Manjaro, but it sounds like the app is starting at least. Our OS detection seems to fail, and should have shown you instructions to install https://libimobiledevice.org. Please try to follow https://libimobiledevice.org/#get-started prior to launching Inspect, and see if that solves the problem for you.

**Comment by ralyodio** [comment]
[youtube-upload](https://www.youtube.com/watch?v=Q4C9zI34X6A)

**Comment by ralyodio** [comment]
https://www.youtube.com/watch?v=Q4C9zI34X6A

**Comment by FryDay** [comment]
I've just adopted the aur package and updated it. Should be good now.

**Comment by auchenberg** [comment]
@ralyodio Alright. Let's try to see if libmobiledevice works as expected try,
﻿
﻿Open a terminal and run `ideviceinfo` (or follow https://libimobiledevice.org/#examples) with your device plugged in. Does the CLI return information about your device?

**Comment by ralyodio** [comment]
trying to install on manjaro again and getting error:

https://aur.archlinux.org/packages/inspect#comment-859546

```
  0     9    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
curl: (22) The requested URL returned error: 404
==> ERROR: Failure while downloading https://github.com/inspectdev/inspect-release/releases/download/v0.8.0/Inspect-0.8.0-linux-amd64.deb
    Aborting...
 -> error making: inspect
```

**Comment by auchenberg** [comment]
@ralyodio Looks like the unofficial package manager mirror is out of date. Please use http://inspect.dev/download to download the latest Inspect release.

**Comment by FryDay** [comment]
The problem is you guys deleted the 0.8.0 release. What's the reason for that?

**Comment by ralyodio** [comment]
Can someone update the aur package to use latest version? or downgrade to 0.79.1

**Comment by ralyodio** [comment]
I had to delete `~/.cache/yay` to get new version of aur package. I don't know if `yay` has a command to do that. I couldn't find one.

**Comment by ralyodio** [comment]
Ok so I installed and got as far as this:

So I installed `yay -S libimobiledevice` now what?

https://youtu.be/-ggn8wqmpNo

I installed ifuse and get a list of apps with `ifuse --listapps` (firefox on ios is the one i want). But the inspect gui doesn't seem to do anything.

**Comment by ralyodio** [comment]
So I installed `yay -S libimobiledevice` now what?

**Comment by strom-und-spiele** [comment]
I recently installed libimobiledevice and inspect from the aur. 
- Inspect version: 0.8.2
- Electron: 19.0.3
- OS version and architecture: linux x64 5.10.194-1-MANJARO
- Language: en_US.UTF-8

It didnt put `inspect-app` in the PATH but I found it under `/opt/Inspect/inspect-app`.
everything works on an iPhone XS iOS 17. 
For some reasons (probably not related to arch) it's not finding debuggable apps on an iPhone 8 iOS 16.
